src/main/java/winston/FindCommand.java

```diff
@@ -0,0 +1,23 @@
+package winston;
+
+public class FindCommand extends Command {
+    private String str;
```

Comment:
Might not be necessary since it's the only String field in this class but could give it a more descriptive name